pkgs/development/compilers/idris2/default.nix

```
@@ -39,6 +42,7 @@ stdenv.mkDerivation rec {
     homepage = https://github.com/idris-lang/Idris2;
```

```suggestion
    homepage = "https://github.com/idris-lang/Idris2";
```
I know, this is not a part of your change, but we could fix this while we are at it.
https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md